config.h.cmake

```diff
@@ -548,3 +548,6 @@ typedef ssize_t IO_SSIZE_T;
 #define _unused(x) (void)x
 #endif
 #endif
+
+#define icalassert(...) assert(__VA_ARGS__)
```

Comment:
@winterz Is this the right place for defining those macros?

Reply:
yep

Reply:
we need a rebase. there are conflicts..

Reply:
here it comes